python_src/program_markers/instrumenter.py

```diff
@@ -247,6 +332,7 @@ class InstrumentedProgram(SourceProgram):
     tracked_markers: tuple[Marker, ...] = tuple()
     disable_prefix: ClassVar[str] = "Disable"
     unreachable_prefix: ClassVar[str] = "Unreachable"
+    used_marker_strategies: tuple[MarkerStrategy, ...] = tuple()
```

Comment:
Does is make sense to assert in `__post_init__` that `used_marker_strategies` is not empty?